dotnet/src/Experimental/Process.Runtime.Dapr/Process.Runtime.Dapr.csproj

```diff
@@ -20,13 +20,17 @@
     <Description>Semantic Kernel Process Dapr Runtime. This package is automatically installed by Semantic Kernel Process packages if needed.</Description>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="C:\Users\crickman\source\repos\semantic-kernel1\dotnet\src\InternalUtilities\process\Runtime\EventExtensions.cs" />
+  </ItemGroup>
```

Comment:
Remove absolute path

Reply:
Indeed...hrm, how did that happen? :wink: